.eslintrc.js

```diff
@@ -163,6 +163,8 @@ module.exports = {
         ],
       },
     ],
+    "no-redeclare": "off",
+    "@typescript-eslint/no-redeclare": ["error"],
```

Comment:
To other reviewers: this fixes

<img width="562" alt="Screenshot 2022-03-04 at 19 03 57" src="https://user-images.githubusercontent.com/608862/156817676-a558baca-7efd-4f3d-8cf2-cba48324df9f.png">